static/src/javascripts/projects/common/modules/commercial/dfp/private/PrebidService.js

```diff
@@ -168,13 +168,13 @@ define([
         if (config.switches.googleAnalytics) {
             // Create a named tracker: if we do this, we need a way to share
             // the account ID
-            // ga('create', 'UA-XXXXX-Z', 'auto', 'prebidTracker');
+            // ga('create', config.ga.accountId, 'auto', 'prebidTracker');
 
             pbjs.enableAnalytics({
                 provider: 'ga',
                 options: {
                     global: 'ga',
-    // trackerName: 'prebidTracker',
+                    // trackerName: 'prebidTracker',
```

Comment:
Perhaps you should add an explanation of why this is commented out and under what conditions it will be enabled.